GliaWidgets/Sources/ViewModel/Chat/ChatViewModel+ChoiceCards.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import SalemoveSDK
+
+extension ChatViewModel {
+    func sendChoiceCardResponse(_ option: ChatChoiceCardOption, to messageId: String) {
+
+        guard let option = option.asSingleChoiceOption() else { return }
+        environment.sendSelectedOptionValue(option) { [weak self] result in
+            guard let self = self else { return }
+
+            switch result {
+            case .success(let message):
+                guard
+                    let selection = message.attachment?.selectedOption
+                else { return }
+
+                self.respond(to: messageId, with: selection)
```

Comment:
I have an interesting question :) 
you got the option from the response, however, messageId from params. Perhaps we can make decision to get both from the response, or both from params (and don't need to have one more guard :)

Reply:
These lines moved to separate file from ChatViewModel to address linter warning on class body length. They are not new. I would hold off changing them now since they are not related to the goal of the PR.